### ISSUE TITLE
model for semantic segmentation

### ISSUE THREAD

**Issue by IITBGoutam** [MouseLand/cellpose#1328]
in ithe semantic.py file `def initialize_class_net(nclasses=5, device=torch.device("cuda")):
    net = vit_sam.Transformer(rdrop=0.4).to(device)
    # default model
    net.load_model("models/cpsam8_0_2100_8_402175188", device=device, strict=False, multigpu=False)
    
    # initialize weights for class maps
    ps = 8 # patch size
    nout = 3
    nclasses = 5
    w0 = net.out.weight.data.detach().clone()
    b0 = net.out.bias.data.detach().clone() 
    net.out = nn.Conv2d(256, (nout + nclasses) * ps**2, kernel_size=1).to(device)
    # set weights for background map
    i = 0
    net.out.weight.data[i * ps**2 : (i+1) * ps**2] = -0.5*w0[(nout-1) * ps**2 : nout * ps**2]
    net.out.bias.data[i * ps**2 : (i+1) * ps**2] = b0[(nout-1) * ps**2 : nout * ps**2]
    # set weights for maps to 4 nuclei classes
    for i in range(1, nclasses):
        net.out.weight.data[i * ps**2 : (i+1) * ps**2] = 0.5*w0[(nout-1) * ps**2 : nout * ps**2]
        net.out.bias.data[i * ps**2 : (i+1) * ps**2] = b0[(nout-1) * ps**2 : nout * ps**2]
    net.out.weight.data[-(nout * ps**2) : ] = w0 
    net.out.bias.data[-(nout * ps**2) : ] = b0
    net.W2 = nn.Parameter(torch.eye((nout + nclasses) * ps**2).reshape((nout + nclasses) * ps**2, nout + nclasses, ps, ps), 
                                requires_grad=False)
    net.to(device);
    return net
` the line `net.load_model("models/cpsam8_0_2100_8_402175188", device=device, strict=False, multigpu=False)` loads a model which is not available , or at least i am not able to find it replacing it with cpsam model provided give wierd results or just error can you help me locate the position of this model

**Comment by mrariden** [comment]
closing due to inactivity
